usmqe_tests/api/ceph/test_cluster.py

```diff
@@ -0,0 +1,122 @@
+"""
+REST API test suite - ceph cluster import
+
+"""
+import pytest
+
+from usmqe.api.tendrlapi import cephapi
+
+
+LOGGER = pytest.get_logger('test_cluster', module=True)
+"""@pylatest default
+Setup
+=====
+"""
+
+"""@pylatest default
+Teardown
+========
+
+"""
+
+"""@pylatest api/ceph.cluster_import
+API-ceph: cluster_import
+***************************
+
+.. test_metadata:: author mkudlej@redhat.com
+
+Description
+===========
+
+Positive import Ceph cluster.
+
+"""
+
+
+def test_cluster_import_valid(valid_session_credentials):
+    """@pylatest api/ceph.cluster_import
+        .. test_step:: 1
+
+        Get list of ids of availible nodes.
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                        {
+                ...
+                  {
+                  "fqdn": hostname,
+                  "machine_id": some_id,
+                  "node_id": node_id
+                  },
+                ...
+                        }
+
+                Return code should be **200** with data ``{"message": "OK"}``.
+
+        """
+    api = cephapi.TendrlApiCeph(auth=valid_session_credentials)
+    nodes = api.get_nodes()
+    ceph_nodes = [node["node_id"] for node in nodes["nodes"] if "ceph" in node["tags"]]
```

Comment:
There's another issue/problem what about if there's more than one ceph cluster? Then just check that node has ceph tag is not enough.